### ISSUE TITLE
conda environment no longer installable

### ISSUE THREAD

**Issue by foxBMS** [foxBMS/foxbms-2#11]
There is currently an unresolved bug when trying to create the conda environment on Windows ([conda_env_win32.yaml](https://github.com/foxBMS/foxbms-2/blob/48da3d01bb5a3195defc007cc119764ccb30f9a5/conf/env/conda_env_win32.yaml)).
This bug revealed to happen since ca. 2 weeks; before installing the environment has worked with any issues for more than a year.

Bug: The pip install process exits with an unresolvable dependency error. As all dependencies are exactly pinned, it is currently unknown why this happens.

Once we have figured out a way to work around this, we will post it here.

**Comment by foxBMS** [comment]
The enviroment verification step described described at [3.3. Development Environment Setup](https://iisb-foxbms.iisb.fraunhofer.de/foxbms/gen2/docs/html/v1.5.0/getting-started/software-installation.html#development-environment-setup)  in the documentation, does no longer work if you applied this workaround.

This means, the step [Verifying the miniconda environment](https://iisb-foxbms.iisb.fraunhofer.de/foxbms/gen2/docs/html/v1.5.0/getting-started/software-installation.html#verifying-the-miniconda-environment), i.e., the command
```cmd
C:\Users\vulpes\Documents\foxbms-2>tools\utils\cmd\run-python-script.bat tests\env\packages_test.py -f tests\env\conda_env_win32.json
```
needs to be skipped.

**Comment by foxBMS** [comment]
We have released version [1.5.1](https://github.com/foxBMS/foxbms-2/releases/tag/v1.5.1), that includes a new environment defintion. Now it is again possible to install the environment and the workaround (disabling the environment check) can be removed, i.e., just delete the line
```python
conf.options.IGNORE_ENV_CHECK = True
```
in the `wscript` in the repository root.
